collections/copernicus-dem.yaml

```diff
@@ -13,7 +13,7 @@ Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/dem/)"
 ViewOnEOBrowser: "[EO Browser](https://sentinelshare.page.link/d3v1)"
 Image: copernicus-dem/copernicus-dem.png
 Resolution: 30m and 90m 
-GeographicalCoverage: Land ([more info](https://spacedata.copernicus.eu/web/cscda/dataset-details?articleId=394198)) under "Area of Interest"
+GeographicalCoverage: Land ([more info](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model)) under "Area of Interest"
```

Comment:
With the new link, 'Area of interest'  section no longer exist. I would clarify this to: 

- European and Worldwide coverage ([more info](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model))